chirp/drivers/baofeng_uv17.py

```diff
@@ -290,9 +290,23 @@ class UV17(baofeng_uv17Pro.UV17Pro):
     } vfo;
 
     #seekto 0x4000;
-    struct {
+    struct channelname {
       char name[11];
-    } names[999];
+    };
+
+    struct {
+      struct channelname names[372];
+    } names1;
+
+    #seek 0x4;
+    struct {
+      struct channelname names[372];
+    } names2;
+
+    #seek 0x4;
+    struct {
+      struct channelname names[255];
+    } names3;
```

Comment:
Why put these in structs? That just increases the indirection you need. Just do this:
```
struct channelname names1[372];
#seek 4;
struct channelname names2[372];
#seek 4;
struct channelname names3[255];
```